9.2/Dockerfile

```diff
@@ -24,14 +24,18 @@ EXPOSE 5432
 # to make sure of that.
 RUN rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
     yum -y --setopt=tsflags=nodocs install https://www.softwarecollections.org/en/scls/rhscl/postgresql92/epel-7-x86_64/download/rhscl-postgresql92-epel-7-x86_64.noarch.rpm && \
-    yum -y --setopt=tsflags=nodocs install gettext postgresql92 && \
+    yum -y --setopt=tsflags=nodocs --enablerepo=centosplus install gettext postgresql92 epel-release && \
+    yum -y --setopt=tsflags=nodocs install nss_wrapper && \
     yum clean all && \
+    chmod go+w /var/run/postgresql && \
```

Comment:
No reason, but now I've run into a weird bug. Only happens locally on docker 1.6:
If I don't chmod right after the yum install for /var/lib/pgsql, then I can't create files in it. I've examined the permissions and they are right (notice permissions on pgsql):
```
bash-4.2$ ls -al /var/lib
total 44
drwxr-xr-x 15 root     root     4096 Apr 29 13:58 .
drwxr-xr-x 25 root     root     4096 Apr 29 13:58 ..
drwxr-xr-x  2 root     root     4096 Apr 15 14:29 alternatives
drwxr-xr-x  2 root     root     4096 Jun 10  2014 games
drwxr-xr-x  2 root     root     4096 Jun 10  2014 misc
drwxrwxrwx  4 postgres postgres 4096 Apr 29 13:58 pgsql
drwxr-xr-x  2 root     root     4096 Apr 29 13:55 rpm
drwxr-xr-x  2 root     root     4096 Jun 10  2014 rpm-state
drwxr-xr-x  2 root     root     4096 Mar  6 05:04 selinux
drwxr-xr-x  4 root     root     4096 Apr 15 14:29 systemd
drwxr-xr-x 10 root     root     4096 Apr 29 13:56 yum
bash-4.2$ touch /var/lib/pgsql/testfile
touch: cannot touch '/var/lib/pgsql/testfile': Permission denied
bash-4.2$
```
If I do the chmod right after yum install, AND after I copy over the contrib directory, then I don't see this issue.